app/main/views/platform_admin.py

```diff
@@ -420,17 +420,26 @@ def clear_cache():
     ])
 
     form = ClearCacheForm()
-    form.model_type.choices = [(key, key.replace('_', ' ').title()) for key in CACHE_KEYS]
+
+    form.model_type.choices = [
+        (key, key.replace('_', ' ').title()) for key in CACHE_KEYS
+    ]
 
     if form.validate_on_submit():
-        to_delete = form.model_type.data
+        group_keys = form.model_type.data
+        groups = map(CACHE_KEYS.get, group_keys)
+        patterns = list(itertools.chain(*groups))
 
-        num_deleted = max(
+        num_deleted = sum(
             redis_client.delete_cache_keys_by_pattern(pattern)
-            for pattern in CACHE_KEYS[to_delete]
+            for pattern in patterns
         )
-        msg = 'Removed {} {} object{} from redis'
-        flash(msg.format(num_deleted, to_delete, 's' if num_deleted != 1 else ''), category='default')
+
+        msg = f'Removed {num_deleted} objects ' \
+              f'across {len(patterns)} key formats ' \
+              f'for {", ".join(group_keys)}'
```

Comment:
I’d prefer we didn’t use backslashes because:
- it makes the amount of indentation arbitrary, which is unusual for Python
- renaming the variable means you need to change the indentation of the subsequent lines, which makes for less-clean diffs

```suggestion
        msg = (
            f'Removed {num_deleted} objects ' 
            f'across {len(patterns)} key formats ' 
            f'for {", ".join(group_keys)}'
        )
```

Reply:
Nice! I'd seen the first technique elsewhere but also prefer your suggestion.

I've amended the last commit with it 👍.